test/object_test.rb

```diff
@@ -24,17 +24,17 @@ class ObjectTest < MiniTest::Spec
   it do
     representer.prepare(target).from_object(source)
 
-    _(target.title).must_equal "The King Is Dead"
-    _(target.album.name).must_equal "RUINER"
-    _(target.album.songs[0].title).must_equal "IN VINO VERITAS II"
+    assert_equal target.title, "The King Is Dead"
```

Comment:
flip the params.  Expectation is the first one then the object.

It minitest syntax to generate the correct error messages.

Reply:
Done